fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -1555,7 +1566,13 @@ public Map<Long, TableSnapshotInfo> collectBaseTableSnapshotInfos(MaterializedVi
                 if (table.isView()) {
                     // skip to collect snapshots for views
                 } else if (table.isOlapTable()) {
-                    Table copied = DeepCopy.copyWithGson(table, OlapTable.class);
+                    OlapTable copied = DeepCopy.copyWithGson(table, OlapTable.class);
```

Comment:
Does copyWithGson has a performance bottleneck? 

can we just use TableSnapshotInfo of the table, if it is not enough, we can extend TableSnapshotInfo, put something crititical in TableSnapshotInfo; then compare these stuff in TableSnapshotInfo and the base table to find out whether the base table is mutated by others.

Reply:
Yep. good idea.

Add some comments and time stats here, we can do this later.